.travis.yml

```diff
@@ -1,12 +0,0 @@
-language: c
```

Comment:
Oh, sorry about that. I deleted it because I initially planned on making this for just me and wasn't using it. It didn't occur to me to share it until after I removed it. 

> On Feb 1, 2016, at 10:25, Todd Treece <notifications@github.com> wrote:
> 
> In .travis.yml:
> 
> > @@ -1,12 +0,0 @@
> > -language: c
> @stuthedew why did you remove the travis config?
> 
> —
> Reply to this email directly or view it on GitHub.
>